__tests__/app.test.js

```diff
@@ -76,6 +76,44 @@ describe("/api/articles/:atricle_id", () => {
         });
     });
   });
+  describe("PATCH", () => {
+    test("200 OK: should return an article with the updated votes by article_id", () => {
+      return request(app)
+        .patch("/api/articles/1")
+        .send({ inc_votes: -50 })
+        .expect(200)
+        .then(({ body }) => {
+          expect(body.article.votes).toBe(50);
+        });
+    });
+    test("400 Bad request: if valid input given for artcle id but non existant", () => {
+      return request(app)
+        .patch("/api/articles/999")
+        .send({ inc_votes: -50 })
+        .expect(400)
+        .then(({ body }) => {
+          expect(body.msg).toBe("Bad request");
+        });
+    });
+    test("400 Bad request: if invalid input given for artcle id", () => {
+      return request(app)
+        .patch("/api/articles/hello")
+        .send({ inc_votes: -50 })
+        .expect(400)
+        .then(({ body }) => {
+          expect(body.msg).toBe("Bad request");
+        });
+    });
```

Comment:
You need a test for a `Status 400, incorrect body, e.g. inc_votes property is not a number`. Please replace the 422 test with this 💡